src/multidim/intervalbox.jl

```diff
@@ -17,6 +17,16 @@ IntervalBox(x...) = IntervalBox(x)
 IntervalBox(x) = IntervalBox(x...)
 IntervalBox(X::IntervalBox, n) = foldl(×, Iterators.repeated(X, n))
 
+# construct from two vectors giving bottom and top corners
```

Comment:
It is more appropriate to have the function `force_interval` in `intervals/interval.jl`. 

I also think the returned intervals should use `Interval` instead of `interval`, since the latter checks (again) if the actual interval is valid.

Reply:
I moved `force_interval` as you suggested, and added tests.

It has to check validity for cases like `force_interval(3, NaN)`.

Reply:
Thanks for the clarification!